script/testing/reporting/parsers/parse_data.py

```diff
@@ -128,13 +131,10 @@ def parse_db_metadata():
     unknown result so the tests won't fail if this happens"""
     regex = r"NOISEPAGE_VERSION[=\s].*(\d.\d.\d)"
     CURRENT_DIR = os.path.dirname(os.path.realpath(__file__))
-    db_metadata = {
-        'noisepage': {
-            'db_version': UNKNOWN_RESULT
-        }
-    }
+    db_metadata = {'noisepage': {'db_version': UNKNOWN_RESULT}}
     try:
-        with open(CURRENT_DIR + '/../../../../src/include/common/version.h') as version_file:
+        with open(CURRENT_DIR +
+                  '/../../../../src/include/common/version.h') as version_file:
```

Comment:
At some point this is going to break if we ever move this file. I would add a `FIXME` here

Reply:
Yah. The good news is we currently don't use this data for anything and it will gracefully fail. I think we should add a comment in the version.h file as well so people know the impact when they make the change.

Reply:
- Added a `FIXME`
- Changed the path concatenation to `os.path.join`